unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/TextShape/Handler/ECSTextShapeComponentHandler.cs

```diff
@@ -199,18 +197,16 @@ internal void ApplyModelChanges(IDCLEntity entity, PBTextShape model)
         {
             textComponent.fontSharedMaterial.EnableKeyword("OUTLINE_ON");
             textComponent.outlineWidth = model.OutlineWidth;
-            if (model.OutlineColor != null)
-            {
-                var outlineColor  = new UnityEngine.Color(model.OutlineColor.R, model.OutlineColor.G, model.OutlineColor.B, 1);
-                textComponent.outlineColor = outlineColor;
-            }
+            var modelOutlineColor = model.GetOutlineColor();
+            var outlineColor  = new UnityEngine.Color(modelOutlineColor.R, modelOutlineColor.G, modelOutlineColor.B, 1);
+            textComponent.outlineColor = outlineColor;
```

Comment:
We should check here if an outline color exist in the model, if it exist we apply it, if not, we skip